README.md

```diff
@@ -30,35 +30,29 @@ Run unit tests with coverage report
 $ yarn coverage
 ```
 
-#### Browser tests
+#### e2e tests
 
-We use Cypress for our browser tests.
-
-Before running cypress, start the development server in another terminal session.
-Ensure that you specify the correct DHIS2_BASE_URL.
+Cypress is used for e2e browser tests. This automatically runs on CI for PRs, the result can be seen on the [Cypress dashboard](https://dashboard.cypress.io/projects/sojh88/). To run the tests locally, define the following in a local `cypress.env.json` file, e.g.:
 
 ```sh
-> DHIS2_BASE_URL=http://localhost:8080 yarn start
-```
-
-There are plans to make this configurable. In order to run the tests, you need to define three environment variables:
-
-```
-CYPRESS_LOGIN_URL=http://localhost:8080 # This must match DHIS2_BASE_URL
-CYPRESS_DHIS2_USERNAME=myusername
-CYPRESS_DHIS2_PASSWORD=mypassword
+{
+    "dhis2_base_url": "https://debug.dhis2.org/dev",
```

Comment:
Sounds like Austin wanted to keep this server private-ish. Suggest changing this to localhost:8080

Reply:
Fair enough, keep in mind that the debug url is still listed further down on the readme though: https://github.com/dhis2/data-visualizer-app/commit/7c3dd5aa75248a58c5a974e932db57218a28a707#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L92, thoughts @amcgee ?